### PR TITLE
Add PPC64LE UI Support

### PR DESCRIPTION
**Pull request by lex-ibm**
GUI builds of VSCodium for PPC64LE would use binaries from [lex-ibm/electron-ppc64le-build-scripts](https://github.com/lex-ibm/electron-ppc64le-build-scripts).

### CODE REVIEW

**Comment by darkyzhou** [comment]
Nice work! BTW, I notice you may have missed updating the following code:

- https://github.com/VSCodium/vscodium/blob/72fc7b746e4f70cb10859228ec1262fd41090a3f/check_tags.sh#L608
- `release_notes.txt`
- Supported Platforms in `README.md`

**Comment by lex-ibm** [comment]
@darkyzhou thanks for the heads up! I've just uploaded the binaries for [v32.2.6](https://github.com/lex-ibm/electron-ppc64le-build-scripts/releases/tag/v32.2.6) and deleted that variable.

File: .github/workflows/insider-linux.yml [hunk 1]
```diff
@@ -174,6 +174,9 @@ jobs:
           vscode_arch: loong64
           npm_arch: loong64
           image: vscodium/vscodium-linux-build-agent:trixie-loong64
+        - slug: PPC64
+          vscode_arch: ppc64le
+          npm_arch: ppc64
```
Comment (daiyam) [inline .github/workflows/insider-linux.yml:179]: @lex-ibm You forgot the image...

Comment (lex-ibm) [reply .github/workflows/insider-linux.yml:179]: Oh! Sorry about that!

**Comment by daiyam** [comment]
@lex-ibm The `.tar.gz` is available ;)

**Pull request merged**